libs/newsletters-data-client/src/lib/newsletter-storage/NewsletterStorage.ts

```diff
@@ -14,14 +14,14 @@ export abstract class NewsletterStorage {
 		SuccessfulStorageResponse<NewsletterData> | UnsuccessfulStorageResponse
 	>;
 
-	abstract get(
+	abstract read(
```

Comment:
Good idea to go for CRUD naming convention, but it does beg the question of whether we should do the same with drafts e.g. change getDraftNewsletter to readDraftNewsletter etc?  Or even change getDraftNewsletter to read??

Reply:
👍 I was planning to do that at some point, but didn't want to have too big a diff on one PR - I'll add a health ticket.